lib/spring/test/acceptance_test.rb

```diff
@@ -47,6 +47,14 @@ def assert_failure(command, expected_output = nil)
         assert_output artifacts, expected_output if expected_output
       end
 
+      def assert_not_include(command, not_expected)
```

Comment:
`assert_not_include` is [defined in test::unit](http://www.rubydoc.info/github/test-unit/test-unit/Test%2FUnit%2FAssertions%3Aassert_not_include).

`assert_output` [defined in minitest](http://ruby-doc.org/stdlib-1.9.3/libdoc/minitest/unit/rdoc/MiniTest/Assertions.html#method-i-assert_output) is also overloaded here. Should I change `assert_not_include` to `refute_include` anyway?